package.json

```diff
@@ -13,6 +13,7 @@
   "repository": "expressjs/cors",
   "main": "./lib/index.js",
   "dependencies": {
+    "debug": "^4.3.1",
```

Comment:
I would set this to the same version express uses, or at least one compatible with the version range of this module.